clash_lib/Cargo.toml

```diff
@@ -86,10 +86,10 @@ axum = { version = "0.8", features = ["ws"] }
 tower-http = { version = "0.6", features = ["fs", "trace", "cors"] }
 chrono = { version = "0.4", features = ["serde"] }
 
-tun = { git = "https://github.com/Watfaq/rust-tun.git", rev = "8f7568190f1200d3e272ca534baf8d1578147e18",  features = ["async"] }
-netstack-lwip = { git = "https://github.com/Watfaq/netstack-lwip.git", rev = "2817bf82740e04bbee6b7bf1165f55657a6ed163" }
+tun = { version = "0.7", features = ["async"] }
+netstack-lwip = { git = "https://github.com/eycorsican/netstack-lwip", rev = "c391c32521b783f32a87cbd511b906b68391f645" }
 
-boringtun = { version = "0.6", git = "https://github.com/cloudflare/boringtun.git", rev = "f672bb6c1e1e371240a8d151f15854687eb740bb" }
+boringtun = { version = "0.6", git = "https://github.com/kubicek/boringtun.git", rev = "e3eb773003c8f2f127cd7ba0407e524e51aa3455" }
```

Comment:
@VendettaReborn i think you mentioned bortingtun/noise fork, maybe it's a good time to do so?